test/units/TEST-74-AUX-UTILS.coredump.sh

```diff
@@ -225,3 +227,32 @@ systemd-run -t --property CoredumpFilter=default ls /tmp
 (! coredumpctl dump --output=/dev/null --output=/dev/null "$CORE_TEST_BIN")
 (! coredumpctl debug --debugger=/bin/false)
 (! coredumpctl debug --debugger=/bin/true --debugger-arguments='"')
+
+# Test for EnterNamespace= feature
+if pkgconf --atleast-version 0.192 libdw ; then
+    # dwfl_set_sysroot() is supported only in libdw-0.192 or newer.
+    cat > "$MAKE_STACKTRACE_DUMP" <<END
+#!/bin/bash
+mount -t tmpfs tmpfs /tmp
+gcc -xc -O0 -g -o $CORE_STACKTRACE_TEST_BIN - <<EOF
+void baz(void) { int *x = 0; *x = 42; }
+void bar(void) { baz(); }
+void foo(void) { bar(); }
+int main(void) { foo(); return 0;}
+EOF
+$CORE_STACKTRACE_TEST_BIN
+END
+    chmod +x "$MAKE_STACKTRACE_DUMP"
+
+    mkdir -p /run/systemd/coredump.conf.d/
+    printf '[Coredump]\nEnterNamespace=no' >/run/systemd/coredump.conf.d/99-enter-namespace.conf
+
+    unshare --pid --fork --mount-proc --mount --uts --ipc --net /bin/bash -c "$MAKE_STACKTRACE_DUMP" || :
+    coredumpctl -1 info "$CORE_STACKTRACE_TEST_BIN" | grep -zvqE 'baz.*bar.*foo'
```

Comment:
I think this is racy. The journal might be slower in writing out the log and then this `coredumpctl` will fail. Maybe insert a sleep or check somehow?